DakeAndDevileCorps/Screens/Store/VC/OperationTimeAndProductTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  OpeerationTimeAndProductTableViewCell.swift
+//  DakeAndDevileCorps
+//
+//  Created by 최동권 on 2022/07/19.
+//
+
+import UIKit
+
+class OperationTimeAndProductTableViewCell: UITableViewCell {
+    static let identifier = "OperationTimeAndProductTableViewCell"
+    @IBOutlet weak var isOperation: UILabel!
+    @IBOutlet weak var todayOperationTime: UILabel!
+    @IBOutlet weak var products: UILabel!
+    override func awakeFromNib() {
```

Comment:
p3;

property들과 함수가 enter없이 이어져있어서 읽기 어려운 거 같아요! 사이사이에 enter를 쳐주면 좋을 듯 합니다.

```suggestion
    static let identifier = "OperationTimeAndProductTableViewCell"
    
    @IBOutlet weak var isOperation: UILabel!
    @IBOutlet weak var todayOperationTime: UILabel!
    @IBOutlet weak var products: UILabel!
    
    override func awakeFromNib() {
```

Reply:
`OperationTimeAndProductTableViewCell.swift` 이 친구 원래 없어야될 친구인데 남아있어서 조용히 보내드렸습니다..
요것도 reference 삭제 이슈였습니다!